ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.10__mce.yaml

```diff
@@ -0,0 +1,56 @@
+base_images:
+  coreos-centos-stream8:
+    name: stream8
+    namespace: coreos
+    tag: "8"
+  openshift_release_golang-1.21:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
```

Comment:
Why is this still 1.19 ?

Reply:
why not? :D

Reply:
As discussed, we have a lot a things that comes with some history. Let's tackle these issues in a follow-up issue